helm/alfresco-process-application/values.yaml

```diff
@@ -291,7 +291,12 @@ alfresco-digital-workspace-app:
   enabled: false
   nameOverride: digital-workspace-app
   ingress:
+    enabled: true
     path: "/{{ .Release.Name }}/digital-workspace"
+    annotations:
+      kubernetes.io/ingress.class: nginx
+      nginx.ingress.kubernetes.io/enable-cors: "true"
+      nginx.ingress.kubernetes.io/cors-allow-headers: Authorization, Content-Type, Accept
```

Comment:
is this line really needed?

as per [docs](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#enable-cors), `Authorization` and `Content-Type` are already in the defaults, while `Accept` is allowed by [specs](https://developer.mozilla.org/en-US/docs/Glossary/CORS-safelisted_request_header)

Reply:
spec doesn't loook that affirmative about `Accept` header. Looks to me it can depend on actual field value too https://fetch.spec.whatwg.org/#cors-safelisted-request-header

@EquinoxBlack do we know what the value of the accept field can be?

Reply:
@alxgomz not really, I was referencing the spec here https://github.com/Alfresco/alfresco-process-infrastructure-deployment/blob/develop/helm/alfresco-process-infrastructure/values.yaml#L506 cc @gionn

Reply:
then do you mind trying without setting the `cors-allow-headers` annotation? Just to make sure we're not allowing dodgy requests by-passing the additional restriction the spec enforces?

Reply:
yep, right now trying to test without it. Will get back here when finished

Reply:
> Looks to me it can depend on actual field value too https://fetch.spec.whatwg.org/#cors-safelisted-request-header

didn't notice that before, however it's more oriented to avoid that invalid values for that field return an affirmative cors response, I think it will be fine removing cors-allow-headers directive

Reply:
@alxgomz all passed without 'cors-allow-headers' annotation. Updated pr